source/guide/index.md

```diff
@@ -6,99 +6,156 @@ order: 1.1
+You can read *[Front-End Frameworks](./front-end-frameworks.html)* to learn more details.
 
 ## Create Your Own App
 
-Weex offered a command line tool, [weex-toolkit](), to help developers to get start easily.
+> The following steps assume you have already known the foundational knowledge of Node.js and npm. If you are not familiar with them, you can visit [https://docs.npmjs.com/](https://docs.npmjs.com/) to learn more about npm.
+
+Weex provide a command line tool, the [weex-toolkit](http://weex-project.io/tools/toolkit.html), to help developers to get start easily. It can help you create a starter project, setup iOS and Android development environments, debug, install plugins and so on.
+
+Currently, the `weex-toolkit` only supports to create the Vue.js project. The `rax-cli` maybe helpful if you want to use Rax. Please visit [Rax's official website](https://alibaba.github.io/rax/) for more details.
 
-### Setup
+### Set up
 
 Make sure you have already installed the [Node.js](https://nodejs.org/), and then install the `weex-toolkit` globally.
 
 ```bash
 npm install weex-toolkit -g
 ```
 
-It will add a `weex` command to your global path, run `weex --help` can see the manual of all its sub commands. The most useful command is `weex create [project-name]`:
+It will add the `weex` command to your global path. Then you can use `weex create` to create a starter project:
 
 ```bash
 weex create awesome-app
 ```
 
-After doing that, you'll get a standard **Weex + Vue.js** project under the `awesome-app` folder.
-
-> Currently, the `weex-toolkit` only supports to create Vue.js project. If you want to use Rax, the `rax-cli` maybe helpful, please go to [Rax's official website](https://alibaba.github.io/rax/) for more details.
+After doing that, you'll get a standard **Weex + Vue.js** project under the `awesome-app` folder of current path.
 
 ### Develop
 
+The next step is get into the path, install dependencies, and start.
 
 ```bash
 cd awesome-app
 npm install
-npm run start
+npm start
 ```
 
-It will start a web server on the `8080` port. You can open `http://localhost:8080/index.html` to see the result rendered on the web. Besides, You can also open the `http://localhost:8080/preview.html` to preview the web render result in iframe and with a QR code generated on the right, which you can use the [Weex plaground app]() to scan it to see the render result on the mobile.
+It will start a web server on the `8081` port.
+
+You can open `http://localhost:8081` to see the render result on the web. Besides, You can also open `http://localhost:8081/preview.html` to preview the web render result in iframe and with a QR code generated on the right. Use the [Weex playground app](http://weex-project.io/playground.html) to scan it and you'll see the render result on the phone.
```

Comment:
to preview the rendered result for Web in iframe.
You can also scan the QR code generated on the right using the Weex playground app then you'll see the rendered result on the mobile device.